content/docs/use-cases/versioning-data-and-model-files/tutorial.md

```diff
@@ -1,8 +1,15 @@
-# Tutorial: Data & Model Versioning
+---
+title: 'Data and Model Versioning Tutorial'
+description: 'Get hands-on experience with data versioning in a basic machine
+learning version control scenario: managing multiple datasets and ML models
+using DVC.'
+---
+
+# Data and Model Versioning Tutorial
```

Comment:
If for reason having "Tutorial" first was intended as a SEO improvement, please open another quick PR for that. Thanks

Reply:
Yes, as mentioned in #1857 that was the intent. Will wait on consensus there and open a new PR if needed.

Reply:
OK please reapply your change in a small PR or throw it in with the next SEO optimization PR you open. Thanks